src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCharClass.cs

```diff
@@ -264,6 +264,137 @@ internal sealed partial class RegexCharClass
+            new LowerCaseMapping('\u0460', '\u0480', LowercaseBor, 0),
+            new LowerCaseMapping('\u0490', '\u04BE', LowercaseBor, 0),
+            new LowerCaseMapping('\u04C1', '\u04C3', LowercaseBad, 0),
+            new LowerCaseMapping('\u04C7', '\u04C7', LowercaseSet, 0x04C8),
+            new LowerCaseMapping('\u04CB', '\u04CB', LowercaseSet, 0x04CC),
+            new LowerCaseMapping('\u04D0', '\u04EA', LowercaseBor, 0),
+            new LowerCaseMapping('\u04EE', '\u04F4', LowercaseBor, 0),
+            new LowerCaseMapping('\u04F8', '\u04F8', LowercaseSet, 0x04F9),
+            new LowerCaseMapping('\u0531', '\u0556', LowercaseAdd, 48),
+            new LowerCaseMapping('\u10A0', '\u10C5', LowercaseAdd, 48),
+            new LowerCaseMapping('\u1E00', '\u1EF8', LowercaseBor, 0),
+            new LowerCaseMapping('\u1F08', '\u1F0F', LowercaseAdd, -8),
+            new LowerCaseMapping('\u1F18', '\u1F1F', LowercaseAdd, -8),
+            new LowerCaseMapping('\u1F28', '\u1F2F', LowercaseAdd, -8),
+            new LowerCaseMapping('\u1F38', '\u1F3F', LowercaseAdd, -8),
+            new LowerCaseMapping('\u1F48', '\u1F4D', LowercaseAdd, -8),
+            new LowerCaseMapping('\u1F59', '\u1F59', LowercaseSet, 0x1F51),
+            new LowerCaseMapping('\u1F5B', '\u1F5B', LowercaseSet, 0x1F53),
+            new LowerCaseMapping('\u1F5D', '\u1F5D', LowercaseSet, 0x1F55),
+            new LowerCaseMapping('\u1F5F', '\u1F5F', LowercaseSet, 0x1F57),
+            new LowerCaseMapping('\u1F68', '\u1F6F', LowercaseAdd, -8),
+            new LowerCaseMapping('\u1F88', '\u1F8F', LowercaseAdd, -8),
+            new LowerCaseMapping('\u1F98', '\u1F9F', LowercaseAdd, -8),
+            new LowerCaseMapping('\u1FA8', '\u1FAF', LowercaseAdd, -8),
+            new LowerCaseMapping('\u1FB8', '\u1FB9', LowercaseAdd, -8),
+            new LowerCaseMapping('\u1FBA', '\u1FBB', LowercaseAdd, -74),
+            new LowerCaseMapping('\u1FBC', '\u1FBC', LowercaseSet, 0x1FB3),
+            new LowerCaseMapping('\u1FC8', '\u1FCB', LowercaseAdd, -86),
+            new LowerCaseMapping('\u1FCC', '\u1FCC', LowercaseSet, 0x1FC3),
+            new LowerCaseMapping('\u1FD8', '\u1FD9', LowercaseAdd, -8),
+            new LowerCaseMapping('\u1FDA', '\u1FDB', LowercaseAdd, -100),
+            new LowerCaseMapping('\u1FE8', '\u1FE9', LowercaseAdd, -8),
+            new LowerCaseMapping('\u1FEA', '\u1FEB', LowercaseAdd, -112),
+            new LowerCaseMapping('\u1FEC', '\u1FEC', LowercaseSet, 0x1FE5),
+            new LowerCaseMapping('\u1FF8', '\u1FF9', LowercaseAdd, -128),
+            new LowerCaseMapping('\u1FFA', '\u1FFB', LowercaseAdd, -126),
+            new LowerCaseMapping('\u1FFC', '\u1FFC', LowercaseSet, 0x1FF3),
+            new LowerCaseMapping('\u2160', '\u216F', LowercaseAdd, 16),
+            new LowerCaseMapping('\u24B6', '\u24D0', LowercaseAdd, 26),
+            new LowerCaseMapping('\uFF21', '\uFF3A', LowercaseAdd, 32),
+        };
```

Comment:
In the name of keeping the change as small as possible, would it be better to just copy/paste this table over the existing one in .MappingTable.cs?  If memory serves, there are only a handful of lines that actually differ.

Reply:
Might as well do this now that a clean revert is not possible. Will push an update shortly

Reply:
Done.